policyengine_us/parameters/gov/irs/deductions/itemized/interest/mortgage/cap.yaml

```diff
@@ -1,14 +1,27 @@
 description: IRS limits the total home mortgage interest deduction to this amount, based on filing status.
+# Debt incurred before December 16, 2017
 SINGLE:
-  2018-01-01: 1_000_000
+  2017-12-16: 1_000_000
 JOINT:
-  2018-01-01: 1_000_000
+  2017-12-16: 1_000_000
 SEPARATE:
-  2018-01-01: 500_000
+  2017-12-16: 500_000
 HEAD_OF_HOUSEHOLD:
-  2018-01-01: 1_000_000
+  2017-12-16: 1_000_000
 SURVIVING_SPOUSE:
-  2018-01-01: 1_000_000
+  2017-12-16: 1_000_000
+
```

Comment:
If we want to add parameter values for multiple years we have to do it in the following way:

```
SINGLE:
  2017-12-16: 1_000_000
  2018-01-01: 750_000
JOINT:
  2017-12-16: 1_000_000
  2018-01-01: 750_000
...
```

Reply:
Also why 12-16?